src/redux/missions/missionsSlice.js

```diff
@@ -0,0 +1,41 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+export const fetchMissions = createAsyncThunk('missions/fetchMissions', async () => {
+  const response = await fetch('https://api.spacexdata.com/v3/missions');
+  const data = await response.json();
+  return data;
+});
```

Comment:
- [ ] The missions data should be filtered to remove unnecessary data. Kindly take a look at [the requirement here](https://github.com/microverseinc/curriculum-react-redux/blob/main/group-project/project_space_travelers_hub.md#redux-fetch-data-and-update-redux-store) to see the data that should be stored.

  ![image](https://github.com/otmaneechchafyky/space-travelers-hub/assets/36542986/ffc75689-2456-442a-8ae4-b0f720ab8b42)

Reply:
fixed